### ISSUE TITLE
目前可以使用自己的数据集进行去噪功能吗

### ISSUE THREAD

**Issue by yl718** [VinAIResearch/WaveDiff#15]

**Comment by hao-pt** [comment]
Yes, you can. Just need to define your own dataloader, you can use `datasets_prep/stackmnist_data.py` for reference.

**Comment by yl718** [comment]
Thank you for your reply. Could you please ask where the input channel for images with noise during testing is? Also, where is the storage path for the output results set!
